.changes/v2023.7.25.md

```diff
@@ -0,0 +1,6 @@
+## [July 25, 2023](https://github.com/OpsLevel/cli/compare/v2023.7.19...v2023.7.25)
```

Comment:
you went too far with changie - on the tools you only need to do `changie new` the `change batch` and `change merge` is fully automated during the release process of the tools.

Reply:
Fixed this